03-task-manager/starter/controller/tasks.js

```diff
@@ -0,0 +1,24 @@
+const Task = require('../models/Task');
+
+exports.getAllTasks = async (req, res) => {
+  const tasks = await Task.find();
+
+  res.json(tasks);
+};
+
+exports.createTask = async (req, res) => {
+  const task = await Task.create(req.body);
+  res.status(201).json(task);
+};
+
+exports.getTask = (req, res) => {
+  res.send('get single task');
```

Comment:
In the video I believe he also has you test out that the URL parameter is coming through by doing:

```javascript
exports.getTask = (req, res) => {
  res.json({ id: req.params.id });
}
```

You're going to be expanding on these functions in the week 6 assignment so no need to change anything here; just wanted to note that in case you didn't get a chance to try it out. 

So the above will return an object with the key `id` and a value that's whatever you put in the URL where it says ":id" in  `/api/v1/tasks/:id`. Eg if you type in `localhost:3000/api/v1/tasks/123` in your browser, it'll return `{ id: 123}`